src/frontend/src/components/Posts/Posts.jsx

```diff
@@ -1,37 +1,217 @@
-import React from 'react';
+import React, { useState, useEffect, useCallback, useRef } from 'react';
 import PropTypes from 'prop-types';
-import { Container } from '@material-ui/core';
+import { Container, Button, Grid } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
-
+import parse from 'parse-link-header';
+import useSiteMetaData from '../../hooks/use-site-metadata';
 import Post from '../Post/Post.jsx';
+import CustomizedSnackBar from '../SnackBar/SnackBar.jsx';
+import Spinner from '../Spinner/Spinner.jsx';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) => ({
   root: {
     padding: 0,
     maxWidth: '785px',
   },
-});
+  content: {
```

Comment:
I'm glad we're moving the posts relating styling directly to the component.

Reply:
Yes, the Header.jsx was another PR #1148. I was the one who made it. 
I thought the #1148 was already merged and closed.

Reply:
I need to teach you how to `rebase` in git to deal with this, but in general, you want to keep all your changes isolated (on their own branches) from each other.